lang/semgrep-grammars/src/semgrep-julia/grammar.js

```diff
@@ -69,23 +65,18 @@ module.exports = grammar(base_grammar, {
       // tree stays the same. Otherwise, we will fail `tree-sitter` tests.
       alias(prec(999,
         $.semgrep_extended_metavariable
-        ), $.identifier)
+      ), $.identifier)
     ),
 
     _expression: ($, previous) => choice(
       previous,
       $.semgrep_ellipsis,
-        $.deep_expression
+      $.deep_expression
     ),
 
     _statement: ($, previous) => choice(
       previous,
       $.semgrep_ellipsis,
     ),
-
-    typed_parameter: ($, previous) => choice(
```

Comment:
Why removing this? We can still have ellpisis in type parameters even with this removal?